contents/blog/new-pricing.md

```diff
@@ -0,0 +1,144 @@
+- You want to track API or server events, especially at high volumes. 
+- You run a large e-commerce or content website with millions of users.
+- You're not tracking a high-volume event because it would be prohibitively expensive.
+- You want to use [autocapture](/docs/product-analytics/autocapture) to track events, but have turned it off to save money.
+
+## How do I start using it?
+
+For most people, it only requires a simple config change to include `process_persons: "identified_only"` in the initialization of your JavaScript Web SDK or snippet.
+
+This will track anonymous traffic without person profiles, and start collecting person information any time an identifying action is taken (i.e. using `identify()`, `group()`, setting person properties with `$set`, etc).
+
+You can get more details on this and other use cases, like server-side events, in our [persons documentation](/docs/data/persons).
+
+## What's the downside?
+
+When sending events without person information, you cannot:
+
+- Filter on persons (e.g. an individual user)
+- Create cohorts filtered on person properties (e.g. paying vs non-paying users)
+- Target by person properties for feature flags, A/B tests, and surveys.
+- Receive initial UTM values for tracking marketing campaign performance (only the most recent).
+- View a person’s profile in the app, or query the persons table in our SQL insights.
+- Use [groups functionality](/docs/product-analytics/group-analytics) on standard events (without person profiles).
+
+If you need the above functionality for all your traffic, you don't need to change anything. Continue to send person information with your events.
+
+Person profiles are not all-or-nothing. You can use them on any traffic where you need more detailed information, and skip them when you don't need that detail. Read more about this in our [persons documentation](/docs/data/persons).
+
+## Why are we doing this?
+
+The short answer? Because we can.
+
+The slightly longer answer? Because it's the right thing to do for both our users and our business.
+
+Most of our competitors are inefficient. They employ huge [outbound sales teams](/founders/negotiate-software-better) to grow revenue. Their salaries and commissions for closing deals are passed onto their customers through higher prices.
+
+In contrast, we're 100% inbound, we grow mostly through word of mouth, and we charge based on actual usage. 
+
+We don't believe in loss-leaders, so we make a modest positive margin on each event sold, but what we charge is directly connected to what it costs us, not what we think we get away with charging.
+
+We grow our revenue through helping you grow, and onboarding you onto other tools, like session replay, feature flags, and surveys as well.
+
+We think this is better than trying to squeeze you for every cent you have. You have a great experience and we'll enjoy better retention and word of mouth for doing the right thing.
+
+> ## FAQ
```

Comment:
TO DO: Add explanation for why person profiles are enabled by default. @raquelmsmith would you mind having a got at this?

Reply:
```suggestion
> ## FAQ

#### Why are person profiles collected if I don't set the config?

For backwards compatibility. We couldn't alter existing customer configs in case this change would mess up how they are tracking things. However, we have plans to change this default in the future - but we didn't want to delay getting these cost saving out until we had that working. For now, all our documented snippets include the `identified_only` config so it's abundantly clear for new and existing users how to set it up.

```